great_expectations/datasource/misc_types.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from abc import ABC
+from dataclasses import dataclass
+
+from great_expectations.types import SerializableDictDot
+
+logger = logging.getLogger(__name__)
+
+
+class BatchIdentifierException(BaseException):
+    # !!! Do we want to create a class for this? Is this the right name and inheritance?
+    pass
+
+class BatchIdentifiers(dict, SerializableDictDot):
+    pass
+
+
+class PassthroughParameters(dict, SerializableDictDot):
+    pass
+    """
+    !!! For full feature parity with previous versions, this will need to support these 5 (or maybe 4) parameters.
+    batch_filter_parameters : {
+        "airflow_run_id": "string_airflow_run_id_that_was_provided",
+        "other_key": "string_other_key_that_was_provided",
+    limit: 10
+    index: Optional[Union[int, list, tuple, slice, str]],  # examples: 0; "-1"; [3:7]; "[2:4]"
```

Comment:
this could be better represented by a self-documenting class

Reply:
Agreed. This was just copied from elsewhere in the codebase.